### DISCUSSION TITLE
Load nerd font icon with five digit code

### DISCUSSION THREAD

**Discussion by baduhai** [JanDeDobbeleer/oh-my-posh#3383] [Troubleshoot] [answered]
How would I go about loading a nerd font icon that has five digits for its hex code? If I try something like "\uf1105" it only reads the first four characters.

**Comment by JanDeDobbeleer** [comment, marked as answer]
I have a solution to get the correct notation. Copy the icon from the nerd font cheat sheet, then paste that into the text field here -> https://convertcodes.com/utf16-encode-decode-convert-string/

Also, see https://github.com/ryanoasis/nerd-fonts/issues/1059
